golang-github-osbuild-composer.spec

```diff
@@ -190,7 +190,12 @@ The worker for osbuild-composer
 %systemd_post osbuild-worker@.service osbuild-remote-worker@.service
 
 %preun worker
-%systemd_preun osbuild-worker@.service osbuild-remote-worker@.service
+# systemd_preun uses systemctl disable --now which doesn't work well with template services.
+# The following lines mimicks its behaviour by running two commands:
```

Comment:
Is there an upstream bug for this?

Reply:
There's now: https://github.com/systemd/systemd/issues/15620

Reply:
And such a detailed one! Thanks :)